### PR TITLE
feat(vuetify): use vuetify module

### PR DESCRIPTION
**Pull request by takehiro-takahashi**
I want to change it to use vuetify files for nuxt.
I thought it would be easier for developers to manage one plug-in.

- [x] Change to vuetify module for nuxt
- [x] Add error layout

### CODE REVIEW

File: template/nuxt/nuxt.config.js [hunk 1]
```diff
@@ -91,6 +87,22 @@ module.exports = {
     // See https://github.com/nuxt-community/axios-module#options
   },<% } %>
 
+  <% if (ui === 'vuetify') { %>
+    /*
+    ** vuetify module configuration
```
Comment (pi0) [inline template/nuxt/nuxt.config.js]: Can you please add a link to the docs here? https://github.com/nuxt-community/vuetify-module

File: template/frameworks/vuetify/layouts/error.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  layout: 'empty',
+  props: {
+    error: {
+      type: Object,
+      default: null
+    }
+  },
+  head () {
+    const title = this.error.statusCode === 404
+      ? pkg.name + ' | ' + this.pageNotFound
```
Comment (pi0) [inline template/frameworks/vuetify/layouts/error.vue]: Maybe we can use `titleTemplate` and avoid importing `pkg`

Comment (takehiro-takahashi) [reply template/frameworks/vuetify/layouts/error.vue]: Thank you !
I fix this problem.

File: template/nuxt/nuxt.config.js (hunk 1 above)
Comment (takehiro-takahashi) [reply template/nuxt/nuxt.config.js]: Thank you!
I added this link.

**Comment by robsontenorio** [comment]
Looks great!

File: template/frameworks/vuetify/layouts/error.vue [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<template>
+  <v-app dark>
```
Comment (pi0) [inline template/frameworks/vuetify/layouts/error.vue]: Dark :+1:  :100:

File: template/nuxt/nuxt.config.js [hunk 4]
```diff
@@ -9,7 +9,6 @@ const { resolve } = require('path')
 <% if (!esm) { -%>
 <% if (ui === 'vuetify') { %>const VuetifyLoaderPlugin = require('vuetify-loader/lib/plugin')<% } %>
```
Comment (robsontenorio) [inline template/nuxt/nuxt.config.js]: I think it should be removed. Already handled through vuetify module.

Comment (takehiro-takahashi) [reply template/nuxt/nuxt.config.js]: @robsontenorio
Thank you for the review! It has been deleted!
https://github.com/nuxt/create-nuxt-app/pull/225/commits/2513287084717aeeca136d9663bd175ad2e4967a

File: template/_package.json [hunk 5]
```diff
@@ -43,8 +43,7 @@
     "@adonisjs/shield": "^1.0.4"<% } %><% if (ui === 'bootstrap') { %>,
     "bootstrap-vue": "^2.0.0-rc.11",
     "bootstrap": "^4.1.3"<% } else if (ui === 'vuetify') { %>,
-    "vuetify": "^1.5.5",
-    "vuetify-loader": "^1.2.1"<% } else if (ui === 'bulma') { %>,
+    "@nuxtjs/vuetify": "latest"<% } else if (ui === 'bulma') { %>,
```
Comment (aldarund) [inline template/_package.json]: @pi0 do we want to use latest? I think it should be consistent with all other packages, so either latest for all or specific version for all

Comment (clarkdo) [reply template/_package.json]: Agree, @takehiro-takahashi could you please change it to the latest stable version  ?

**Comment by clarkdo** [comment]
@takehiro-takahashi Could you please fix the conflicts/tests and revert `yarn.lock` ?

**Comment by clarkdo** [comment]
@takehiro-takahashi  Thanks!

**Comment by emelendez** [comment]
Fresh install now doesn't work!!!
\nuxt.config.js
  68:16  error  'colors' is not defined  no-undef
  69:15  error  'colors' is not defined  no-undef
  70:18  error  'colors' is not defined  no-undef
  71:13  error  'colors' is not defined  no-undef
  72:16  error  'colors' is not defined  no-undef
  73:14  error  'colors' is not defined  no-undef
  74:16  error  'colors' is not defined  no-undef

✖ 7 problems (7 errors, 0 warnings)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zf-acs@1.0.0 lint: `eslint --ext .js,.vue --ignore-path .gitignore . "--fix"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the zf-acs@1.0.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

**Comment by clarkdo** [comment]
I’ll propose a fix shortly

**Comment by clarkdo** [comment]
@emelendez v2.7.1 released

**Pull request merged**